conductr_cli/conduct_logging.py

```diff
@@ -112,3 +113,16 @@ def raise_for_status_inc_3xx(response):
     response.raise_for_status()
     if response.status_code >= 300:
         raise HTTPError(status_codes._codes[response.status_code], response=response)
+
+
+def format_timestamp(timestamp, args):
+    date = arrow.get(timestamp)
+
+    if args.date and args.utc:
+        return date.to('UTC').strftime('%Y-%m-%dT%XZ')
+    elif args.date:
+        return date.to('local').strftime('%Y-%m-%dT%X%z')
+    elif args.utc:
+        return date.to('UTC').strftime('%XZ')
+    else:
+        return date.to('local').strftime('%X%z')
```

Comment:
Yes, these times are localised.

The call `arrow.get()` returns an instance of `Arrow` object which has a `to` method. The `to` method accepts 'local' or 'UTC' which will convert to local and UTC date/time respectively: http://arrow.readthedocs.org/en/latest/#arrow.arrow.Arrow.to